targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52832/device/TOOLCHAIN_ARM_STD/nRF52832.sct

```diff
@@ -26,14 +26,10 @@
 LR_IROM1 MBED_APP_START MBED_APP_SIZE {
   ER_IROM1 MBED_APP_START MBED_APP_SIZE {
    *.o (RESET, +First)
-   *(InRoot$$Sections)
-   __start_sdh_soc_observers *(sdh_soc_observers) __stop_sdh_soc_observers
```

Comment:
why are these lines being removed ( __start_xxx) ?

Reply:
Because they are not needed anymore; they were added with the first fix which wasn't reliable enough.